### PR TITLE
[34205] Get list users api

### PR DESCRIPTION
**Pull request by ghost**
Ticket:
[34205](https://edu-redmine.sun-asterisk.vn/issues/34205)

What PR do ?
- Get list users by api controller
- Authentication with JWT
- Write unit test

Impact range
N/A

UT pass
Yes

Ticket bug in backlog
None

Envident:
http://127.0.0.1:3000
image

### CODE REVIEW

**Comment by ghost** [comment]
ready

**Comment by vanvtt-0952** [comment]
commented
Có dám mạnh miệng nói là em có tự review pull trước khi ready ko nào? tui thu 10k liền nè

**Comment by ghost** [comment]
ready

File: app/controllers/api/v1/users_controller.rb [hunk 1]
```diff
@@ -0,0 +1,13 @@
+class Api::V1::UsersController < Api::V1::BaseController
+  before_action :authenticate_request!
+
+  def index
+    @users = User.order("created_at DESC")
```
Comment (vanvtt-0952) [inline app/controllers/api/v1/users_controller.rb]: viết scope đi bé

**Comment by ghost** [comment]
ready

**Pull request merged**